db/schema.rb

```diff
@@ -1143,6 +1146,7 @@
     t.datetime "created_at", precision: nil, null: false
     t.datetime "updated_at", precision: nil, null: false
     t.index ["external_id"], name: "index_webauthn_credentials_on_external_id", unique: true
+    t.index ["nickname", "user_id"], name: "index_webauthn_credentials_on_nickname_and_user_id", unique: true
```

Comment:
What about having the index on `['user_id', 'nickname']` so that we could later drop `index_webauthn_credentials_on_user_id`?